avoid writing past the end of the gcr_track buffer
https://bugs.gentoo.org/show_bug.cgi?id=464708

--- ./src/diskimage/fsimage-create.c.orig
+++ ./src/diskimage/fsimage-create.c
@@ -262,7 +262,7 @@
         gcrptr = gcr_track;
         util_word_to_le_buf(gcrptr, disk_image_raw_track_size(image->type, track));
         gcrptr += 2;
-        memset(gcrptr, 0x55, NUM_MAX_BYTES_TRACK);
+        memset(gcrptr, 0x55, NUM_MAX_BYTES_TRACK - 2);
 
         header.track = track;
         for (sector = 0;
